cli/mender-artifact/debugfs.go

```diff
@@ -110,43 +76,43 @@ func debugfsCopyFile(file, image string) (string, error) {
 func debugfsReplaceFile(imageFile, hostFile, image string) (err error) {
 	// First check that the path exists. (cd path)
 	cmd := fmt.Sprintf("cd %s\nclose", filepath.Dir(imageFile))
-	if _, err = executeCommand(cmd, image); err != nil {
+	if _, err = debugfsExecuteCommand(cmd, image); err != nil {
```

Comment:
I take it my naming on this one made you look twice as to what it actually was :p

Reply:
Haha, yes, exactly.=!